src/platform/packages/shared/kbn-scout/src/playwright/fixtures/worker/scout_space/parallel.ts

```diff
@@ -19,46 +19,42 @@ export const scoutSpaceParallelFixture = coreWorkerFixtures.extend<
 >({
   scoutSpace: [
     async ({ log, kbnClient }, use, workerInfo) => {
-      const spaceId = `test-space-${workerInfo.workerIndex}`;
+      const spaceId = `test-space-${workerInfo.parallelIndex + 1}`;
```

Comment:
`workerIndex` vs `parallelIndex`:

1) Playwright created new worker after failure, `workerIndex` will be increased. Since we delete Kibana space on fixture tear down, we can and should use persistent index  -  `parallelIndex`